packages/grid/src/PivotGrid.tsx

```diff
@@ -155,7 +157,7 @@ export const PivotGrid = React.memo(<Name extends string = string>(props: Props<
   );
   const styleProp = props.style || (emptyObject as PivotGridStyle);
   const borderStyle = styleProp.border || defaultBorderStyle;
-  const cellStyle = styleProp.cell || emptyObject;
+  const cellStyle = isFunction(styleProp.cell) ? styleProp.cell : () => (styleProp.cell || emptyObject);
```

Comment:
```suggestion
  const cellStyle = styleProp.cell || emptyObject;
```

I guess we don't need this change anymore